manifests/i/Intel/BluetoothDriver/23.100.1/Intel.BluetoothDriver.installer.yaml

```diff
@@ -0,0 +1,10 @@
+PackageIdentifier: Intel.BluetoothDriver
+PackageVersion: 23.100.1
```

Comment:
```suggestion
PackageVersion: 23.100.1
InstallerSwitches:
  Silent: /qn
  SilentWithProgress: /qn
```

Above suggestions are instructed by [How to Install Intel® Bluetooth Drivers in Silent Mode](https://www.intel.com/content/www/us/en/support/articles/000088100/wireless.html).

Verified in _Windows Sandbox_.

```text
--> Installing WinGet
--> Disabling safety warning when running installers
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
已启用管理员设置 'LocalManifestFiles'。
已启用管理员设置 'LocalArchiveMalwareScanOverride'。

--> Installing the Manifest 23.100.1

已找到 Intel® Wireless Bluetooth® Drivers [Intel.BluetoothDriver] 版本 23.100.1
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
正在下载 https://downloadmirror.intel.com/843147/BT-23.100.1-64UWD-Win10-Win11.exe
  ██████████████████████████████  70.9 MB / 70.9 MB
已成功验证安装程序哈希
正在启动程序包安装...
已成功安装

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName            DisplayVersion Publisher         ProductCode                            Scope
-----------            -------------- ---------         -----------                            -----
英特尔(R) 无线 Bluetooth(R) 23.100.1.1     Intel Corporation {00001100-0230-2052-84C8-B8D95FA3C8C3} Machine
```

![image](https://github.com/user-attachments/assets/28aedfd9-ac8c-44b4-b577-37a0970c40a7)